tasks/collect_logs.yml

```diff
@@ -0,0 +1,55 @@
+- name: Ensure local logs dir exists
+  connection: local
+  file:
+    state: "{{ item }}"
+    path: "{{ rhv_he_deploy_log_dir }}"
+  # remove then recreate log dir
+  with_items:
+    - absent
+    - directory
+- name: Check if remote logs dir exists
+  stat:
+    path: /var/log/ovirt-hosted-engine-setup
```

Comment:
Do we want this value to be parameterizable? Is it changeable during install? At the least, we should move it out to the `vars/main.yml` file and avoid magic constants.

Reply:
This value is not changeable to the best of my knowledge, but it is still a magic var and should be fixed.

Reply:
Now defined in defaults